dev_guide/managing_images.adoc

```diff
@@ -477,8 +477,9 @@ $ oc secrets new-dockercfg <pull_secret_name> \
     --docker-password=<password> --docker-email=<email>
 ----
 
-To use a secret for pulling images for pods, you must add the secret to your
-service account. The name of the service account in this example should match
+If `serviceAccountConfig.limitSecretReferences` is set to `true` in the master
```

Comment:
The BZ wasn't desperately clear (sorry), but did mention not to review the 'oc secrets link --for=pull' case(s), of which this is one. serviceAccountConfig.limitSecretReferences being enabled affects the requirement to run 'oc secrets link' or 'oc secrets link --for=mount', but not 'oc secrets link --for=pull'.